### DISCUSSION TITLE
Vue Issue after panel work

### DISCUSSION THREAD

**Discussion by Azimuth0r** [emporia-vue-local/esphome#217] [Q&A]
I powered down my panel to do some work, added a Type2 plug on bus surge protector. then turned everything back on. 
I did move the breakers for the Vue but verified both legs have power.
Now I don't get readings from the Vue.

I get the following during boot: 
[08:43:27][W][emporia_vue:067]: Detected 1 missing reading(s), data may not be accurate!

and then the following: 
[08:48:15][W][component:204]: Component ota took a long time for an operation (0.16 s).
[08:48:15][W][component:205]: Components should block for at most 20-30ms.
I'm not sure if any of that is relevant.

**Comment by flaviut** [comment]
If you see nothing but these warnings, that's a problem.

If you only saw them a few times, it's not a big deal. Losing half a second's data out of an hour won't impact anything.

**Reply by Azimuth0r** [reply]
the strange thing is that I'm missing all data on most of the circuits, for example circuit 11 - solar in (there is 4.3KW solar coming in) 
circuit 9 and 10 should be the sub panel for all of the 110v circuits in the house. they show no draw either. 
Circuit 2 does show accurate readings of 3.8kw charging the battery. so not all are non functional.


I don't know if the log numbers in brackets corelate, but 067 is [09:47:11][C][i2c.idf:067]:   Recovery: bus successfully recovered
further up in the logs.

I may have mixed up L1 and L2 in the power feed for the Vue, is it possible that would cause 0W readings?

**Reply by flaviut** [reply]
> I may have mixed up L1 and L2 in the power feed for the Vue, is it possible that would cause 0W readings?

Yes. Remove all the `*pos` entries from your config and you'll see negatives where the phases are reversed or the clamps are reversed.

**Reply by Azimuth0r** [reply]
Yup, that's what I did. thanks for your help.
